alerts/cluster-logging-operator/CollectorHigh403ForbiddenResponseRate.md

```diff
@@ -0,0 +1,62 @@
+# CollectorHigh403ForbiddenResponseRate
+
+## Meaning
+
+The `CollectorHigh403ForbiddenResponseRate` alert indicates that the log collector
+is experiencing a sustained high rate (at least 10% over the last 2 minutes, persisting
+for 5 minutes) of **HTTP 403 Forbidden** responses when attempting to send log
+data to its configured log store.
+
+An **HTTP 403 Forbidden** response indicates that the server understood the request
+but refuses to authorize it.
+
+## Impact
+
+Log collection and forwarding to the destination are impaired,
+resulting in data loss at the configured log store.
+
+## Diagnosis
+
+1. Examine `CollectorHigh403ForbiddenResponseRate` alert details in OpenShift console:
+    - Note the key labels:
+        - `app_kubernetes_io_instance`: The name of the collector.
+        - `namespace`: Namespace of the collector.
+        - `component_id`: The ID of the specific output/sink that's failing. This
+        corresponds to an output defined in your `ClusterLogForwarder` (CLF) custom
+        resource.
+2. Review the CLF Configuration:
+    1. Identify the output definition corresponding to the `component_id` obtained
```

Comment:
I don't recall if we can consistently say the output in question will be named XXXX and you can get it from the id as `output_NAME`.  This would be ideal to mention here